qiskit_experiments/database_service/db_experiment_data.py

```diff
@@ -130,6 +129,36 @@ def __json_encode__(self):
         return self.__getstate__()
 
 
+class FigureData:
```

Comment:
This could probably just be called `Figure`.

Reply:
I would also add a `__repr__` or `_repr_html_` method that prints the name and displays the actual figure (not exactly sure how to do this). This should work whether the figure is a MPL figure, or a loaded SVG from database (later can is IPython.SVG I think).

Reply:
`Figure` is already taken by pyplot

Reply:
Using `Figure` will cause confusion with pyplot's figure; also, this requires changing the way figures are referenced all over the code.